react.js

```diff
@@ -36,6 +36,10 @@ module.exports = function shopifyReactPreset(context, options = {}) {
     }
   }
 
+  if (env !== 'test') {
```

Comment:
In the past, I've needed testIDs in dev mode.  That's based on Capybara/Selenium being pigs to work with, though.

No harm in starting out with only `test` mode and waiting for feedback.